docs/kyma-release-notes/2023-11-08-release-notes-2.20.md

```diff
@@ -0,0 +1,21 @@
+..placeholder for preface..
```

Comment:
Colorful leaves dropping on the ground signify high autumn. With Kyma 2.20, we say goodbye to some of our components that fall down the Kyma tree. However, it is a natural process as they will be reborn in their modular versions and we will watch them in full bloom again. Read on to learn more about changes in the new Kyma release.